src/lib.rs

```diff
@@ -90,8 +91,13 @@ const fn default_max_data_xfer_size() -> u32 {
     1048576
 }
 
-const fn default_migration_capabilities() -> MigrationCapabilities {
-    MigrationCapabilities { pgsize: 4096 }
+#[inline(always)]
+fn pagesize() -> u32 {
+    unsafe { sysconf(_SC_PAGESIZE) as u32 }
+}
+
+fn default_migration_capabilities() -> MigrationCapabilities {
+    MigrationCapabilities { pgsize: pagesize()}
```

Comment:
Commit message title should be: "protocol: Remove hardcoded page size"

Reply:
Hi @rbradford ,I have updated the title.

Reply:
nit: I would prefer a space after the colon.

Reply:
I would like you to update for this nit ^

Reply:
> I would like you to update for this nit ^

@rbradford There is a space after the colon at line 100.Do you mean line 100?

Reply:
@rbradford I added a space after the colon in the title.